### ISSUE TITLE
Any update on the recent downtime?

### ISSUE THREAD

**Issue by derryl** [ewolfe/prlint#115]
Hi, there! Thanks for creating PRLint. It's been very helpful for our team's workflow.

[The service has been down a lot recently](https://stats.uptimerobot.com/ZzYnEf2BW) – and we're having to manually override the affected PR's. Is this a temporary situation? Or should we look into a different solution for linting our PR's?

If there's anything we can to do help (i.e. it's some sort of technical issue) – let me know! 

Cheers

**Comment by ewolfe** [comment]
Yeah this is just a temporary situation. I haven't investigated the issue yet (the past 2 weeks at work have been really busy), but I'm pretty sure I'm missing a simple try/catch somewhere regarding parsing an invalid `.github/prlint.json` file. I'll be able to investigate further this weekend.

P.S. it's back up right now.

I'll keep this issue open until I can put in some safeguards and an auto restart mechanism.

**Comment by derryl** [comment]
Cool, thanks for the quick response!
